Makefile

```diff
@@ -18,3 +18,6 @@ test: install_dependencies
 
 lint: install_dependencies
 	poetry run $(BLACK_CHECK)
+
+typecheck:
+	poetry run mypy .
```

Comment:
Looks like you forgot to add `install_dependencies` here. Look at the lines above for a clue.

Reply:
Thanks, it now installs mypy by adding `install_dependencies`, I was missing that.